### ISSUE TITLE
Create a Personalized Sourcing Strategy with SAP Customer Order Sourcing

### ISSUE THREAD

**Issue by i837683** [SAPDocuments/Issues#33]
Tutorial: https://developers.sap.com/tutorials/cos-getting-started-trial.html
--------------------------

In Step 2: Set up SAP Customer Order Sourcing via Booster, I run through the steps and get this message. The message is mildly useful, except the tutorial doesn't cover what to do in this case. No help is provided, nothing to help me get beyond this point.  I have a trial account, I'm an SAP employee. 

![image](https://user-images.githubusercontent.com/8920618/103694512-ecdf4d00-4f4f-11eb-95d7-0ba70bba6c17.png)

I also go thru the steps in the manual set up and I can't find SAP Customer Order Sourcing (step 4) in Subscriptions.  https://developers.sap.com/tutorials/cos-manual-setup.html

When I select subscriptions under my trial sub account, I can't find Customer Order Sourcing to subscribe to the service. Am I missing something? Shouldn't be this hard to start at trial version, right?

**Comment by mjeann** [comment]
Hi @i837683 I got a ticket for that on my last working day before my vacation (which lasted until today). 
But thank you for reminding me to check this tutorial as well.
I will add the missing information some time this or next week.

Here's how to proceed:
In your Global Account go to **Entitlements > Entity Assignments**.
Choose **Configure Entitlements** and **Add Service Plans**.
Search for "SAP Customer Order Sourcing Trial" and add both available plans.

We found out this step has to be done in case the trial user didn't originally set up a Global Account in EU10.
